pom.xml

```diff
@@ -45,6 +45,7 @@
     <module>kie-dmn-feel</module>
     <module>kie-dmn-backend</module>
     <module>kie-dmn-core</module>
+    <module>kie-dmn-validation</module>
```

Comment:
The module `kie-dmn-validation` is added to the reactor build of the parent `kie-dmn`